### DISCUSSION TITLE
Greenplum Connection Capability with Open Metadata

### DISCUSSION THREAD

**Discussion by GitForPy** [open-metadata/OpenMetadata#12417] [General]
Hi there,

I've been exploring the capabilities of the Open Metadata system for our project and I've come across a question.

I've noticed that PostgreSQL support is included in the Open Metadata connectors. I'm wondering if it's possible to connect to Greenplum? Greenplum plays a key role in our tech stack and it would be great if we could integrate it with Open Metadata.

I would appreciate any information on this matter.

Thanks!

**Comment by astoulov** [comment]
Arenadata Data Catalog that is based on OpenMetadata has connector for GP. But it's not free.
